server/base/src/main/java/org/apache/accumulo/server/fs/FileManager.java

```diff
@@ -94,12 +93,12 @@ public boolean equals(Object obj) {
 
     @Override
     public int hashCode() {
-      return fileName.hashCode();
+      return file.hashCode();
     }
   }
 
-  private Map<String,List<OpenReader>> openFiles;
-  private HashMap<FileSKVIterator,String> reservedReaders;
+  private Map<TabletFile,List<OpenReader>> openFiles;
```

Comment:
Anytime we use TabletFile as key we have to be wary of #2830.  I think this is probably ok for now, but future changes to StoredTabletFile equals and hashcode could break this.

Reply:
Agree that this should be fine in this case (both objects resolve to the same equals anyways) but we will definitely need to make sure things don't break when #2380 is fixed.